misc/install-deps.sh

```diff
@@ -14,20 +14,20 @@ distro=$(grep "^ID=" /etc/os-release | cut -d\= -f2 | sed -e 's/"//g')
 case $distro in
     "ubuntu" | "debian")
         apt-get $OPT install pandoc libdw-dev libpython2.7-dev libncursesw5-dev pkg-config
-        apt-get $OPT install libcapstone-dev ;;
+        apt-get $OPT install libcapstone-dev libluajit-5.1-dev ;;
```

Comment:
ok, please squash this into an earlier commit.

Reply:
Do you mean squashing into c5b83a367b18eda50c4c48250ffd97cb82b835ba and change the commit message ?

Reply:
Fixed